templates/openstack/consulting.html

```diff
@@ -13,7 +13,7 @@ <h2 class="p-heading--three">Carrier grade, banking security, government scale.
         <p>Hundreds of successful OpenStack clouds underpin our consulting, training, architecture design, and hardware procurement advice - to help you evaluate and deliver OpenStack.</p>
         <p>
           <a href="/openstack/contact-us?product=foundation-cloud" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Engage Canonical exports', 'eventLabel' : 'Engage Canonical exports - Top CTA', 'eventValue' : undefined });" class="p-button--positive">Engage Canonical experts</a>
-          <a href="{{ ASSET_SERVER_URL }}6a031fb9-EN_Foundation-Cloud-Build_screen-AW_19.07.18.pdf" class="p-button--neutral">Read our consulting datasheet</a>
+          <a href="{{ ASSET_SERVER_URL }}6e65055c-EN_Foundation-Cloud-Build_screen-AW_07.08.18.pdf" class="p-button--neutral">Read our consulting datasheet</a>
```

Comment:
sorry, but can we rename this to Foundation-Cloud-Build-2018-08.pdf and update the spreadsheet